ios/default.nix

```diff
@@ -278,11 +276,77 @@ nixpkgs.runCommand "${executableName}-app" (rec {
     # focus????
     focus/reflex-platform/scripts/run-in-ios-sim "$tmpdir/${executableName}.app"
   '';
-}) ''
+  portableDeployScript = nixpkgs.writeText "make-portable-deploy" ''
+    #!/usr/bin/env bash
+    set -eo pipefail
+
+    if (( "$#" < 1 )); then
```

Comment:
Maybe use `!=` instead?
I've had to write a few shell-in-nix scripts recently and the "lesser than" eventually made for confusing errors when I had multiple arguments, removed one and forgot to update the invocations, making some args shifted.
That is, assuming the `$@` in https://github.com/reflex-frp/reflex-platform/pull/677/files#diff-249f6c1656767d0e08530aa4eaf1fa33R338 refers to the inner script's arguments

Reply:
Note the PR for that change only is here: https://github.com/reflex-frp/reflex-platform/pull/682